packages/contracts-bedrock/deployments/11155111-deploy.json

```diff
@@ -0,0 +1,7 @@
+{
+  "CheckBalanceLow": "0xd4Da04ae46C54BD747500F3C133FdB8559409E21",
```

Comment:
Should check out the intermediate deployment and place it in `superchain-registry`